packages/inventory/src/api/api.js

```diff
@@ -78,6 +78,35 @@ export function getEntities(items, {
 }, showTags) {
     if (hasItems && items.length > 0) {
         return hosts.apiHostGetHostById(items, undefined, perPage, page, undefined, undefined, { cancelToken: controller && controller.token })
+        .then(async (data) => {
+            let result;
+
+            try {
+                result = await hosts.apiHostGetHostSystemProfileById(
+                    items,
+                    perPage,
+                    page,
+                    orderBy,
+                    undefined,
+                    undefined,
+                    undefined,
+                    {
+                        cancelToken: controller && controller.token,
+                        query: { 'fields[system_profile]': [ 'os_release' ]
```

Comment:
I would either change the `generateFilter` to be named as `queryFromObject` or add another `generateFields` (the function should be generic enough to handle how fields are generated as well)

Reply:
This is so simple I would rather to avoid using external functions from dependencies.

Reply:
Yeah, but I was more thinking about future plans, more specifically if some app will use `getEntities` and they'd want to pull in custom fields (there is already a mock fot this in remediations, cloud-connector and edge)